native/chat/inner-text-message.react.js

```diff
@@ -78,20 +64,11 @@ class InnerTextMessage extends React.PureComponent<Props> {
         onPress={this.props.onPress}
         onLongPress={this.props.onPress}
         activeOpacity={0.6}
+        style={[styles.message, messageStyle, cornerStyle]}
       >
-        <Hyperlink
-          linkDefault={true}
-          style={[styles.message, messageStyle, cornerStyle]}
-          linkStyle={linkStyle}
-        >
-          <Text
-            onPress={this.props.onPress}
-            onLongPress={this.props.onPress}
-            style={[textStyle, textCustomStyle]}
-          >
-            {text}
-          </Text>
-        </Hyperlink>
+        <Markdown onLink={url => Linking.openURL(url)} darkStyle={darkColor}>
```

Comment:
Let's move `onLink` to a static method on the class so the function doesn't get reconstructed on every render